src/App.tsx

```diff
@@ -374,14 +376,14 @@ function ThreadGPT(props: ThreadGPT) {
           </Indent>
         )}
         {mutation.isError && (
-          <Indent depth={data.depth + 1}>
+          <Indent depth={data.depth + 1} margin={margin}>
             <div className="alert alert-danger" role="alert">
               {renderMutationError(mutation.error)}
             </div>
           </Indent>
         )}
         {effectiveShowCreateForm && (
-          <Indent depth={data.depth + 1}>
+          <Indent depth={data.depth + 1} margin={margin}>
```

Comment:
Having to pass `margin={margin}` from the top into multiple components is "prop drilling". It causes re-render inefficiencies and clutters the code. I’ll add a commit to undo this.